src/puzzles.js

```diff
@@ -0,0 +1,362 @@
+let puzzlesArray = [
```

Comment:
Any reason this is `let` instead of `const`? Are we reassigning this anywhere?

Reply:
This can be changed to `const` because no new arrays or values are reassigned to `puzzlesArray` throughout the program. Only the contents of the `puzzlesArray` are being changed inside `shufflePuzzles` function.

I did a quick search and found out...
- We can change the contents of a `const` array, but cannot reassign a new array or a different value.